becquerel/core/fitting.py

```diff
@@ -1440,8 +1435,8 @@ def custom_plot(
             # Add info about the ROI and units
             if self.roi:
                 s += "ROI: [{:.3f}, {:.3f}]\n".format(*self.roi)
-            s += "X units: {:s}\n".format(self.xmode if self.xmode else "None")
-            s += "Y units: {:s}\n".format(self.ymode if self.ymode else "None")
+            s += "X units: {:s}\n".format(self.xmode or "None")
+            s += "Y units: {:s}\n".format(self.ymode or "None")
```

Comment:
I dislike this change---it relies on a subtlety unique to python that not even many intermediate python programmers know, just for the sake of a little brevity. Could we disable this?

Reply:
Your alternate idea: convert both to `f"{str(self.xymode)}"` for maximum conciseness.

Reply:
This is almost certainly [FURB110](https://docs.astral.sh/ruff/rules/if-exp-instead-of-or-operator/)